src/metaphlan.jl

```diff
@@ -43,13 +43,19 @@ end
 
 
 """
-Option1: take a path to merged table (eg test/files/metaphlan_multi_profile.tsv)
+Option1: take a path to merged table (eg test/files/metaphlan_multi_test.tsv)
     and make CommunityProfile
 
-Option2: take vector of paths to single tables (eg ["test/files/metaphlan_single1_profile.tsv", "test/files/metaphlan_single2_profile.tsv"])
+Option2: take vector of paths to single tables (eg ["test/files/metaphlan_single1.tsv", "test/files/metaphlan_single2.tsv"])
     and make CommunityProfile
 """
-function metaphlan_profiles(tables)
+function metaphlan_profiles(path::AbstractString; level=:all)
+    profiles = CSV.read(path, DataFrame)
+    taxa = [last(_split_clades(c)) for c in profiles[:, "#SampleID"]]
+    mat = sparse(Matrix(profiles[:, 2:end]))
+    samples = MicrobiomeSample.(first.(split.(names(profiles[:, 2:end]), "_")))
+    keep = level == :all ? Colon() : [ismissing(c) || c == level for c in clade.(taxa)]
```

Comment:
Do we want to return rows that don't have a clade here?

I kinda think we need a second argument, like `keep_unidentified' or something to trigger that. I would set the default to `false`.

Also, ultimately, the function to filter on a particular taxonomic level should live in `Microbiome.jl` - I've opened an issue: https://github.com/BioJulia/Microbiome.jl/issues/67

Reply:
I just added a `keepunidentified=false` keyword argument. Does that work?

Reply:
Yeah, but doesn't look like you actually use it in the logic of this line.... The idea is that `[ismissing(c) || c == level for c in clade.(taxa)]` will keep things that are missing, regardless of the value of `keepunidentified`...

Reply:
I just edited the function to actually use `keepunidentified` but I'm still not sure if I wrote it correctly.